src/main/java/online/talkandtravel/service/impl/ChatServiceImpl.java

```
@@ -193,6 +203,18 @@ private Chat createAndSavePrivateChat(User user, User companion) {
             .build());
   }
 
+  private Chat createAndSaveSubChatWithUser(NewSubChatRequest request, User user) {
+    Country country = getCountry(request.countryId());
```

you should check if country exists in database. for this purpose fetch country by id, throw if country not found. Make it in private method

---

actually, this method already exists , so just use it

---

> you should check if country exists in database. for this purpose fetch country by id
For this, I've used getCountry private method, if a country is not found an exception will be thrown 

> actually, this method already exists
Did you mean getCountry method?

---

yes)))